### ISSUE TITLE
VIZ: draw_data improvements 

### ISSUE THREAD

**Issue by martin-henz** [source-academy/cadet-frontend#475]
```
const x = y => y; draw_data(pair(x, pair(x,x)));
```
Currently we get: 
<img width="158" alt="screenshot 2019-03-07 at 5 58 07 pm" src="https://user-images.githubusercontent.com/30220768/53948356-96f6af80-4102-11e9-9569-506ba1a5c0e7.png">
but it should actually show *1 in all three slots, to indicate co-references.

**Comment by martin-henz** [comment]
Asking @lumos309  if we should postpone and close the issue.

**Comment by martin-henz** [comment]
Consolidating issues: moving this one here: https://github.com/source-academy/cadet-frontend/issues/452

**Comment by martin-henz** [comment]
TL;DR: summary of draw_data improvement suggestions:

(1) Co-references to functions in draw_data (two-eyes display of functions)

(2)

**Comment by martin-henz** [comment]
TL;DR: summary of draw_data improvement suggestions:

(1) Write quotation marks around strings, see above.

(2) draw_list(my_array) displays *1 instead of an array. It should show arrays, using a syntax similar to pairs: they should be horizontally arranged boxes.

**Comment by martin-henz** [comment]
Currently the tool does not distinguish between true and "true":

<img width="539" alt="Screenshot 2019-07-10 at 9 09 57 PM" src="https://user-images.githubusercontent.com/30220768/60972297-92643100-a358-11e9-880c-bf1d421eacc0.png">

<img width="539" alt="Screenshot 2019-07-10 at 9 09 38 PM" src="https://user-images.githubusercontent.com/30220768/60972318-9a23d580-a358-11e9-803c-d4ce972dcc08.png">

We should put quotation marks around strings.

**Comment by seanlowjk** [comment]
We should revisit this if we have the time for it

**Comment by martin-henz** [comment]
These issues are fixed. Can close.

<img width="524" alt="Screenshot 2020-05-21 at 10 22 30 AM" src="https://user-images.githubusercontent.com/30220768/82516391-2a23d580-9b4d-11ea-9269-a1bb510de9e7.png">

<img width="712" alt="Screenshot 2020-05-21 at 10 26 38 AM" src="https://user-images.githubusercontent.com/30220768/82516535-9e5e7900-9b4d-11ea-86ef-57d9ab09f60a.png">
